chsdi/models/vector/uvek.py

```diff
@@ -807,6 +807,47 @@ class StatistikwasserkraftanlagenNew(Base, Vector):
 register('ch.bfe.statistik-wasserkraftanlagen', StatistikwasserkraftanlagenNew)
 
 
+class Erneuerbarheizen(Base, Vector):
+    __tablename__ = 'erneuerbarheizen'
+    __table_args__ = ({'schema': 'bfe', 'autoload': False})
+    __template__ = 'templates/htmlpopup/erneuerbarheizen.mako'
+    __bodId__ = 'ch.bfe.erneuerbarheizen'
+    __queryable_attributes__ = ['company']
+    __label__ = 'company'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    xtf_id = Column('xtf_id', Integer)
+    firstname = Column('firstname', Unicode)
+    name = Column('name', Unicode)
+    company = Column('company', Unicode)
+    firstname_name = Column('firstname_name', Unicode)
+    email = Column('email', Unicode)
+    phonenumber = Column('phonenumber', Unicode)
+    street = Column('street', Unicode)
+    streetnumber = Column('streetnumber', Unicode)
+    street_streetnumber = Column('street_streetnumber', Unicode)
+    postalcode = Column('postalcode', Integer)
+    place = Column('place', Unicode)
+    pc_place = Column('pc_place', Unicode)
+    website = Column('website', Unicode)
+    privatecontrol = Column('privatecontrol', Unicode)
+    additionalinformation = Column('additionalinformation', Unicode)
+    consulting_type_de = Column('consulting_type_de', Unicode)
```

Comment:
do we really need these attributes in the model / identify result? they're not referenced in the mako.

Reply:
yes, the bfe need all the attributes in the API...